### PR TITLE
Features/test2

### PR DESCRIPTION
**Pull request by Wahaj-Ali**
in this branch
- added test for the following functions:
   1) edit task
   2) check task status
   3) clear completed tasks
- also added a test for the mockDocumnet.js
- since I am using jest version> 24.x, so I did not need to mock local storage. Hence, no test for local storage mock object exists.

### CODE REVIEW

File: test/tests/addtask.test.js [hunk 1]
```diff
@@ -57,4 +58,57 @@ describe('This test', () => {
       3,
     );
   });
-});
\ No newline at end of file
+
+  test('edits the 1st task description', () => {
```
Comment (Olamarx) [inline test/tests/addtask.test.js]: - [x] Kindly group them using the description() method. This makes your test organized and separate concerns.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Wahaj-Ali, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- All linters passed ✔️ 
- Test passed ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test/tests/addtask.test.js (hunk 1 above)
Comment (Wahaj-Ali) [reply test/tests/addtask.test.js]: @Olamarx Thank you for your kind suggestions. I am going to implement them😄.

**Review by gemmen29 — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**